src/views/docs-view/index.tsx

```diff
@@ -63,12 +64,14 @@ const DocsView = ({ mdxSource, lazy }: DocsViewProps) => {
   return (
     <DevDotContent>
       <NoIndexTagIfVersioned />
-      <MDXRemote
-        compiledSource={compiledSource}
-        components={components}
-        lazy={lazy}
-        scope={scope}
-      />
+      <TabProvider>
```

Comment:
I've added `TabProvider` for `docs-view` and `tutorial-view` content specifically.